src/bilan/importerEmpreinte.usecase.ts

```diff
@@ -0,0 +1,13 @@
+import { EmpreinteRepository } from "@/bilan/ports/empreinteRepository";
+
+export class importerEmpreinteUsecase {
+  private _empreinteRepository: EmpreinteRepository;
+
+  constructor(empreinteRepository: EmpreinteRepository) {
+    this._empreinteRepository = empreinteRepository;
+  }
+
+  async execute(idNGC: string, utilisateurId: string): Promise<boolean> {
```

Comment:
Tu n'es pas obligé d'utiliser async/await et tu faire directement `    return this._empreinteRepository.importerSituationNGC(idNGC, utilisateurId);`